samples/bluetooth/peripheral_dis/src/main.c

```diff
@@ -44,6 +45,57 @@ static struct bt_conn_cb conn_callbacks = {
 	.disconnected = disconnected,
 };
 
+
```

Comment:
Redundant empty line (there should never be a need to have two consecutive ones)

Reply:
OK